src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -1686,6 +1686,7 @@ namespace Akka.Actor
         public TerminatedProps() { }
         public override Akka.Actor.ActorBase NewActor() { }
     }
+    [System.ObsoleteAttribute("TypedActor in its current shape will be removed in v1.5")]
```

Comment:
Since there are multiple ways to replace TypedActor (the simplest one is using `Handle((dynamic)message)`), I think we can just give some deadline expectation to the users. Version 1.5 will already have breaking changes (change of default serializer, potentially more of them), we could remove this class then.